bigtable/client/testing/embedded_server_test_fixture.cc

```diff
@@ -0,0 +1,71 @@
+// Copyright 2018 Google Inc.
```

Comment:
s/Inc./LLC/ we changed this boilerplate recently.  No need to change existing files.

Reply:
fixed